recipes/argtable2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "argtable2" %}
+{% set version = "2.13" %}
+{% set md5 = "156773989d0d6406cea36526d3926668" %}
```

Comment:
conda-forge prefers sha256. In this case, `'8f77e8a7ced5301af6e22f47302fdbc3b1ff41f2b83c43c77ae5ca041771ddbf'`